### PR TITLE
Feature/ot plat rssi

### PR DESCRIPTION
**Pull request by mszczodrak**

### CODE REVIEW

File: include/platform/radio.h [hunk 1]
```diff
@@ -274,7 +274,7 @@ extern void otPlatRadioTransmitDone(bool aFramePending, ThreadError aError);
  *
  * @returns The noise floor value in dBm when the noise floor value is valid.  127 when noise floor value is invalid.
```
Comment (jwhui) [inline include/platform/radio.h]: Replace `noise floor` with `RSSI`.

File: examples/platforms/cc2538/radio.c [hunk 2]
```diff
@@ -300,7 +300,7 @@ RadioPacket *otPlatRadioGetTransmitBuffer(void)
     return &sTransmitFrame;
 }
 
-int8_t otPlatRadioGetNoiseFloor(void)
+int8_t otPlatRadioGetRssi(void)
 {
     return 0;
```
Comment (darconeous) [inline examples/platforms/cc2538/radio.c]: If you aren't returning the actual value, return 127 instead, to indicate that the returned value is invalid.

We may want to add a constant for it though. `kOtUndefinedPower`, perhaps?

**Comment by darconeous** [comment]
👍

**Comment by abtink** [comment]
LGTM :+1:

**Comment by jwhui** [comment]
LGTM 👍

**Pull request merged**